src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 3;
+
+  return (x) => {
+    return (y) => {
+      counter--;
+
+      if (counter > 0 || counter % 2 === 0) {
+        return x + y;
+      }
+
+      return 'Bzzz... Error!';
```

Comment:
You can avoid `if` statement here and use only ternary operation

Reply:
also agree, ternary operation looks better here